### ISSUE TITLE
Security issue detected by ckeditor

### ISSUE THREAD

**Issue by zsoltjanes** [waynestate/nova-ckeditor4-field#95]
Hi! Can you please take a look about this message,which appeared to me from today on the ckeditor.

![image](https://github.com/waynestate/nova-ckeditor4-field/assets/7995436/7c02d39b-78c9-4d06-b785-c5ca36223f04)

**Comment by Hirannuwanpriya** [comment]
I'm also getting the Same Warning.

**Comment by Hirannuwanpriya** [comment]
I added **'versionCheck' => false,** to the Nova config file for the time being, and the warning has disappeared."

**Comment by zsoltjanes** [comment]
> Adding `'versionCheck' => false` to `config/nova/ckeditor-field.php` didn't work for me. I also don't see any code references to that property.

Add it to the options section

**Comment by PieterCappelle** [comment]
Came here for the same issue :-) (thanks for the solution!)

**Comment by chrispelzer** [comment]
Hey all!

I just released [v1.4.0](https://github.com/waynestate/nova-ckeditor4-field/releases/tag/1.4.0) due to it defaulting the version checking to be false by default. I didn't want to check people who have been using the LTS off guard.

Thanks to @adriandmitroca for the PR. I also added some information to the top of the README.md about what this is and if you wish to get LTS from CKEditor4.

It's out of my hands as all this package is providing the integration with Laravel Nova.

The best thing would be to move to CKEditor5. I've been exploring it in my projects and these days I can't tell the difference between it and CKeditor4 anymore. The thing that I really needed CKEditor4 was the ability to edit source, at the start of CKEditor5 that wasn't the case.

There is a CKEditor5 package for Laravel Nova at https://github.com/mostafaznv/nova-ckeditor which last was updated 2 months, so they are still going strong with their CKEditor5 package.
